post-scan-actions/aws-python-email-notification/handler.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+import boto3
+from botocore.exceptions import ClientError
+
+def lambda_handler(event, context):
+    
+    sender = os.environ['SENDER']
+    recipient = os.environ['RECIPIENT']    
+    aws_region = os.environ['AWS_REGION']
+    subject = os.environ['SUBJECT']
+    charset = "UTF-8"
+        
+    for record in event['Records']:
+        # Analyze the message and look for findings
+        message = json.loads(record['Sns']['Message'])         
+        findings = message['scanning_result'].get('Findings')
+
+        if findings:
+            # The email body for recipients with HTML and non-HTML email clients.
+            body_text = '''\
+                CloudOne FSS Email Notification
+                File URL: {file_url}
+                Malware: {malware}
+                Type: {type}\
+                '''.format(file_url=str(message['file_url']),
+                           malware=str(message['scanning_result']['Findings'][0].get('malware')),
+                           type=str(message['scanning_result']['Findings'][0].get('type')))
+            body_html = '''\
+                <html><head></head><body><h1>CloudOne FSS Email Notification</h1><p>
+                <p><b>File URL: </b>{file_url}</p>
+                <p><b>Malware: </b>{malware}</p>
+                <p><b>Type: </b>{type}</p>
+                </html></body></p>\
+                '''.format(file_url=str(message['file_url']),
+                           malware=str(message['scanning_result']['Findings'][0].get('malware')),
+                           type=str(message['scanning_result']['Findings'][0].get('type')))
```

Comment:
`Findings` is an array. Unless you on purpose only want to use the first finding, please loop through the array.

Reply:
You're right. I thought, why should a finding have more than one malware type? Since it's an array I changed the code.